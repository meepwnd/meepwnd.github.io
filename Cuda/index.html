<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="description" content="This webpage is made by Sergey Bombin using a free PSD template for demonstration purposes only.">
	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
	<title>Cuda</title>
	<link href="https://fonts.googleapis.com/css?family=Titillium+Web:300,400,400i,600,700" rel="stylesheet">
	<link rel="stylesheet" href="css/main.css">
	<script defer src="https://use.fontawesome.com/releases/v5.0.10/js/all.js" integrity="sha384-slN8GvtUJGnv6ca26v8EzVaR9DC58QEwsIk9q1QXdCU8Yu8ck/tL/5szYlBbqmS+" crossorigin="anonymous"></script>
</head>
<body>
	<!-- NAVIGATION -->
	<nav class="nav" id="nav">
		<div class="container nav__container">
			<a href="#" class="nav__logo">
				<img src="images/logo.png" alt="Cuda">
			</a>
			<ul class="nav__menu">
				<li class="nav__item">
					<a class="nav__link" href="#scrollspy-home">home</a>
				</li>
				<li class="nav__item"><a class="nav__link" href="#scrollspy-about">About</a></li>
				<li class="nav__item"><a class="nav__link" href="#scrollspy-work">Work</a></li>
				<li class="nav__item"><a class="nav__link" href="#scrollspy-blog">Blog</a></li>
				<li class="nav__item"><a class="nav__link" href="#scrollspy-contact">contact</a></li>
			</ul>

			<div class="nav-trigger">
				<span class="bar1"></span>
				<span class="bar2"></span>
				<span class="bar3"></span>
			</div>
		</div>
	</nav>

	<!-- HERO -->
	<section class="hero" id="scrollspy-home">
		<div class="container">
			<h1 class="hero__heading">Hi there! We are the new kids on the block 
			and we build awesome websites and mobile apps.</h1>
			<a href="#" class="btn btn-warning">WORK WITH US!</a>
		</div>
	</section>

	<!-- SERVICES -->
	<section class="services" id="scrollspy-about">
		<div class="container">
			<h2 class="services_heading">SERVICES WE PROVIDE</h2>
			<h5>We are working with both individuals and businesses from all over the globe <br>
			to create awesome websites and applications.</h5>
			<div class="row">
				<div class="col-md-6 col-lg-3 services__block">
					<div class="icon">
						<i class="icon-flag"></i>
					</div>
					<h3 class="services__subheading">Branding</h3>
					<p class="services__text">Lorem ipsum dolor sit amet, 
					consectetuer adipiscing elit, sed diam nonummy nibh.</p>
				</div>
				<div class="col-md-6 col-lg-3 services__block">
					<div class="icon">
						<i class="icon-pencil"></i>
					</div>
					<h3 class="services__subheading">DESIGN</h3>
					<p class="services__text">Sed ut perspiciatis unde omnis iste natus error sit voluptatem  </p>
				</div>
				<div class="col-md-6 col-lg-3 services__block">
					<div class="icon">
						<i class="icon-cogs"></i>
					</div>
					<h3 class="services__subheading">DEVELOPMENT</h3>
					<p class="services__text">At vero eos et accusamus et iusto odio dignissimos qui blanditiis praesentium.</p>
				</div>
				<div class="col-md-6 col-lg-3 services__block">
					<div class="icon">
						<i class="icon-rocket"></i>
					</div>
					<h3 class="services__subheading">ROCKET SCIENCE</h3>
					<p class="services__text">Et harum quidem rerum est et expedita distinctio. Nam libero tempore.</p>
				</div>
			</div>
		</div>
	</section>

	<!-- TEAM -->
	<section class="team-section">
		<div class="container">
			<h2 class="team-section__heading">MEET OUR BEAUTIFUL TEAM</h2>
			<h5 class="team-section__subheading">We are a small team of designers and developers, 
			who help brands with big ideas.</h5>
			<div class="row team">
				<div class="col-md-6 col-lg-3 team__box">
					<img src="images/icon-team.png" alt="ANNE HATHAWAY">
					<h3 class="team__name">ANNE HATHAWAY</h3>
					<h6 class="team__position">CEO / Marketing Guru</h6>
					<p class="team__text">Lorem ipsum dolor sit amet, 
					consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna.</p>
					<div class="contacts">
						<a href="#" class="contacts__link">
							<i class="fab fa-facebook-f"></i>
						</a>
						<a href="#" class="contacts__link">
							<i class="fab fa-twitter"></i>
						</a>
						<a href="#" class="contacts__link">
							<i class="fab fa-linkedin-in"></i>
						</a>
						<a href="#" class="contacts__link">
							<i class="fas fa-envelope"></i>
						</a>
					</div>
				</div>
				<div class="col-md-6 col-lg-3 team__box">
					<img src="images/icon-team.png" alt="Kate Upton">
					<h3 class="team__name">Kate Upton</h3>
					<h6 class="team__position">Creative Director</h6>
					<p class="team__text">Duis aute irure dolor in in voluptate velit esse cillum dolore fugiat nulla pariatur. Excepteur sint occaecat non diam proident.</p>
					<div class="contacts">
						<a href="#" class="contacts__link">
							<i class="fab fa-twitter"></i>
						</a>
						<a href="#" class="contacts__link">
							<i class="fab fa-linkedin-in"></i>
						</a>
						<a href="#" class="contacts__link">
							<i class="fas fa-envelope"></i>
						</a>
					</div>
				</div>
				<div class="col-md-6 col-lg-3 team__box">
					<img src="images/icon-team.png" alt="Olivia Wilde">
					<h3 class="team__name">Olivia Wilde</h3>
					<h6 class="team__position">Lead Designer</h6>
					<p class="team__text">Nemo enim ipsam voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem nesciunt.</p>
					<div class="contacts">
						<a href="#" class="contacts__link">
							<i class="fab fa-facebook-f"></i>
						</a>
						<a href="#" class="contacts__link">
							<i class="fab fa-twitter"></i>
						</a>
						<a href="#" class="contacts__link">
							<i class="fab fa-linkedin-in"></i>
						</a>
						<a href="#" class="contacts__link">
							<i class="fas fa-envelope"></i>
						</a>
					</div>
				</div>
				<div class="col-md-6 col-lg-3 team__box">
					<img src="images/icon-team.png" alt="Ashley Greene">
					<h3 class="team__name">Ashley Greene</h3>
					<h6 class="team__position">SEO / Developer</h6>
					<p class="team__text">Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam.</p>
					<div class="contacts">
						<a href="#" class="contacts__link">
							<i class="fab fa-facebook-f"></i>
						</a>
						<a href="#" class="contacts__link">
							<i class="fab fa-twitter"></i>
						</a>
						<a href="#" class="contacts__link">
							<i class="fas fa-envelope"></i>
						</a>
					</div>
				</div>
			</div>
		</div><!-- container end -->
	</section>

	<!-- SKILLS -->
	<section class="skills-section container">
		<h2 class="skills-section__heading">WE GOT SKILLS!</h2>
		<h5>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod <br>
		tempor incididunt ut labore et dolore magna aliqua.</h5>
		<div class="row skills">
			<div class="col-sm-6 col-md-3 skills__box">
				<svg class="radial-progress blue" data-percentage="90" viewBox="0 0 80 80">
				  <circle class="incomplete" cx="40" cy="40" r="35"></circle>
				  <circle class="complete" cx="40" cy="40" r="35" style="stroke-dashoffset: 39.58406743523136;"></circle>
				  <text class="percentage" x="47%" y="61%" transform="matrix(0, 1, -1, 0, 80, 0)">90</text>
				  <text class="percentage-sign" x="69%" y="60%" transform="matrix(0, 1, -1, 0, 80, 0)">%</text>
				</svg>

				<h3 class="skills__heading">Web Design</h3>
			</div>
			<div class="col-sm-6 col-md-3 skills__box">
				<svg class="radial-progress purple" data-percentage="75" viewBox="0 0 80 80">
				  <circle class="incomplete" cx="40" cy="40" r="35"></circle>
				  <circle class="complete" cx="40" cy="40" r="35" style="stroke-dashoffset: 39.58406743523136;"></circle>
				  <text class="percentage" x="47%" y="61%" transform="matrix(0, 1, -1, 0, 80, 0)">75</text>
				  <text class="percentage-sign" x="69%" y="60%" transform="matrix(0, 1, -1, 0, 80, 0)">%</text>
				</svg>
				<h3 class="skills__heading">HTML / CSS</h3>
			</div>
			<div class="col-sm-6 col-md-3 skills__box">
				<svg class="radial-progress lightblue" data-percentage="70" viewBox="0 0 80 80">
				  <circle class="incomplete" cx="40" cy="40" r="35"></circle>
				  <circle class="complete" cx="40" cy="40" r="35" style="stroke-dashoffset: 39.58406743523136;"></circle>
				  <text class="percentage" x="47%" y="61%" transform="matrix(0, 1, -1, 0, 80, 0)">70</text>
				  <text class="percentage-sign" x="69%" y="60%" transform="matrix(0, 1, -1, 0, 80, 0)">%</text>
				</svg>
				<h3 class="skills__heading">GRAPHIC DESIGN</h3>
			</div>
			<div class="col-sm-6 col-md-3 skills__box">
				<svg class="radial-progress orange" data-percentage="85" viewBox="0 0 80 80">
				  <circle class="incomplete" cx="40" cy="40" r="35"></circle>
				  <circle class="complete" cx="40" cy="40" r="35" style="stroke-dashoffset: 39.58406743523136;"></circle>
				  <text class="percentage" x="47%" y="61%" transform="matrix(0, 1, -1, 0, 80, 0)">85</text>
				  <text class="percentage-sign" x="69%" y="60%" transform="matrix(0, 1, -1, 0, 80, 0)">%</text>
				</svg>
				<h3 class="skills__heading">UI / UX</h3>
			</div>
		</div>
	</section>

	<!-- PORTFOLIO -->
	<section class="portfolio-section" id="scrollspy-work">
		<h2 class="portfolio-section__heading">OUR PORTFOLIO</h2>
		<h5 class="portfolio-section__subheading">Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet
		consectetur, adipisci velit, sed quia non numquam</h5>
		<div class="container portfolio">
			<div class="row">
				<div class="col-md-6 portfolio__item">
					<img src="images/image1.png" alt="">
					<h5 class="portfolio__title">Isometric Perspective Mock-Up</h5>
				</div>
				<div class="col-md-6 portfolio__item">
					<img src="images/image2.png" alt="">
					<h5 class="portfolio__title">Time Zone App UI</h5>
				</div>
			</div>
			<div class="row">
				<div class="col-md-6 portfolio__item">
					<img src="images/image3.png" alt="">
					<h5 class="portfolio__title">Viro Media Players UI</h5>
				</div>
				<div class="col-md-6 portfolio__item">
					<img src="images/image4.png" alt="">
					<h5 class="portfolio__title">Blog / Magazine Flat UI Kit</h5>
				</div>
			</div>
			<div class="row">
				<div class="col-md-6 portfolio__item" style="display:none">
					<img src="images/image1.png" alt="">
					<h5 class="portfolio__title">Isometric Perspective Mock-Up</h5>
				</div>
				<div class="col-md-6 portfolio__item" style="display:none">
					<img src="images/image2.png" alt="">
					<h5 class="portfolio__title">Time Zone App UI</h5>
				</div>
			</div>
			<div class="row">
				<div class="col-md-6 portfolio__item" style="display:none">
					<img src="images/image3.png" alt="">
					<h5 class="portfolio__title">Isometric Perspective Mock-Up</h5>
				</div>
				<div class="col-md-6 portfolio__item" style="display:none">
					<img src="images/image4.png" alt="">
					<h5 class="portfolio__title">Time Zone App UI</h5>
				</div>
			</div>
			<a href="#" id="loadMore" class="btn btn-load">LOAD MORE PROJECTS</a>
		</div><!-- container end -->
	</section>

	<!-- TESTIMONIALS -->
	<section class="testimonials-section" id="scrollspy-blog">
		<div class="container">
			<h2 class="testimonials-section__heading">WHAT PEOPLE SAY ABOUT US</h2>
			<h5 class="testimonials-section__subheading">Our clients love us!</h5>
			<div class="row testimonials">
				<div class="col-md-6 testimonials__box testimonials__box_top">
					<div class="testimonials__image-wrapper">
						<img src="images/icon-client.png" alt="" class="testimonials__image">
					</div>
					<i class="testimonials__feedback">“Nullam dapibus blandit orci, viverra gravida dui lobortis eget. Maecenas fringilla urna eu nisl scelerisque.”</i>
					<h3 class="testimonials__name">Chanel Iman</h3>
					<h5 class="testimonials__position">CEO of Pinterest</h5>
				</div>
				<div class="col-md-6 testimonials__box testimonials__box_top">
					<div class="testimonials__image-wrapper testimonials__image-wrapper_even">
						<img src="images/icon-client.png" alt="" class="testimonials__image">
					</div>
					<i class="testimonials__feedback">“Vivamus luctus urna sed urna ultricies ac tempor dui sagittis. In condimentum facilisis porta.”</i>
					<h3 class="testimonials__name">ADRIANA LIMA</h3>
					<h5 class="testimonials__position">Founder of Instagram</h5>
				</div>
			</div>
			<div class="row">
				<div class="col-md-6 testimonials__box">
					<div class="testimonials__image-wrapper">
						<img src="images/icon-client.png" alt="" class="testimonials__image">
					</div>
					<i class="testimonials__feedback">“Vivamus luctus urna sed urna ultricies ac tempor dui sagittis. In condimentum facilisis porta.”</i>
					<h3 class="testimonials__name">ANNE HATHAWAY</h3>
					<h5 class="testimonials__position">Lead Designer at Behance</h5>
				</div>
				<div class="col-md-6 testimonials__box">
					<div class="testimonials__image-wrapper testimonials__image-wrapper_even">
						<img src="images/icon-client.png" alt="" class="testimonials__image">
					</div>
					<i class="testimonials__feedback">“Phasellus non purus vel arcu tempor commodo. Fusce semper, purus vel luctus molestie, risus sem cursus neque.”</i>
					<h3 class="testimonials__name">EMMA STONE</h3>
					<h5 class="testimonials__position">Co-Founder of Shazam</h5>
				</div>
			</div>
		</div><!-- container end -->
	</section>

	<!-- CONTACT -->
	<section class="contact" id="scrollspy-contact">
		<div class="container">
			<h2 class="contact__heading">GET IN TOUCH</h2>
			<h5 class="contact__subheading" x-ms-format-detection="none">1600 Pennsylvania Ave NW, Washington, DC 20500, United States of America. Tel: (202) 456-1111</h5>
			<form action="" class="contact-form" id="form">
				<div class="field-wrapper">
					<div class="field field_left">
						
						<input type="text" class="contact-form__name" 
						id="name" placeholder="Your Name *" name="name">
					</div><!--
						
					--><div class="field field_right">
						<input type="email" class="contact-form__email" 
						id="email" placeholder="Your Email *" name="email">
					</div>
				</div>
				
					
				<div class="field">
					<textarea name="message" id="" class="contact-form__message" id="message" placeholder="Your Message *"></textarea>
				</div>
					
				<input type="submit" class="btn btn-contact" value="SEND MESSAGE">
			</form>
		</div>
	</section>

	<!-- FOOTER -->
	<footer class="footer">
		<div class="container">
			<ul class="social-nav">
				<li class="social-nav__item">
					<a href="#" class="social-nav__link">Facebook</a>
				</li>
				<li class="social-nav__item">
					<a href="#" class="social-nav__link">Twitter</a>
				</li>
				<li class="social-nav__item">
					<a href="#" class="social-nav__link">Google+</a>
				</li>
				<li class="social-nav__item">
					<a href="#" class="social-nav__link">Linkedin</a>
				</li>
				<li class="social-nav__item">
					<a href="#" class="social-nav__link">Behance</a>
				</li>
				<li class="social-nav__item">
					<a href="#" class="social-nav__link">Dribbble</a>
				</li>
				<li class="social-nav__item">
					<a href="#" class="social-nav__link">Github</a>
				</li>
			</ul>
		</div>
	</footer>

	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
	<script src="js/jquery.validate.min.js"></script>
	<script src="js/jquery.min.js"></script>

</html>