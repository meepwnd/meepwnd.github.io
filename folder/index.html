<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Document</title>
	<link rel="stylesheet" href="reset.css">
	<link rel="stylesheet" href="style.css">
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>

</head>
<body>
	<header class="container">
		<div class="header-top">
			<a href="#" class="logo">
				<img src="images/logo.png" alt="">
			</a>
			<a href="#" class="menu-trig"></a>

			<form action="">
				<input type="search">
				<input type="submit" value="GO">
			</form>
		</div>
		
		<nav>
			<ul class="menu">
				<li>home</li>
				<li>
					<span>about us</span>
					<ul class="submenu">
						<li>item1</li>
						<li>item2</li>
						<li>item3</li>
					</ul>
				</li>
				<li>services</li>
				<li>partners</li>
				<li>customers</li>
				<li>products</li>
				<li>careers</li>
				<li>contact</li>
			</ul>
		</nav>

	</header>

	<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Cum mollitia dicta vero repudiandae laborum optio consectetur corrupti soluta. Repellendus eius vero impedit quos inventore incidunt, cum velit explicabo, necessitatibus, eum consectetur nostrum omnis pariatur eveniet maxime blanditiis quibusdam reiciendis odit odio libero accusantium cupiditate architecto. Non ducimus nostrum aut quia accusamus officiis beatae sit, reprehenderit molestias, eius, doloremque amet culpa commodi rem quae eos, molestiae. Neque aut quod fuga recusandae earum nisi quae itaque esse culpa dolorum veniam blanditiis, alias doloribus quaerat dicta iure corporis ratione natus odit aspernatur a! Laudantium natus dicta nobis odit quae officia id odio animi quasi nesciunt nisi, perspiciatis harum optio molestias ratione quas magnam inventore culpa rerum fugit minima, ipsa suscipit. Impedit laborum quod nobis recusandae eius? Veritatis officia provident pariatur rerum suscipit laboriosam ea cumque excepturi necessitatibus exercitationem consectetur inventore ipsum eum ducimus, quidem beatae non, molestiae dignissimos similique voluptate, eos nesciunt doloremque aperiam illo. Possimus dolor ullam, exercitationem repellendus animi incidunt laboriosam tenetur eaque quo totam distinctio, nostrum cum culpa repellat dicta dolore nihil, ipsam molestias soluta eligendi? Quam deleniti nobis ea, inventore, a dolorem at mollitia vel atque rem magni exercitationem alias esse qui earum numquam maxime consequatur ut quod magnam.</p>
	<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Natus voluptatibus voluptatum nisi ullam! A facilis, dolorem provident. Voluptates inventore ipsum, enim veniam nihil deleniti. Rem molestiae, necessitatibus quos repudiandae possimus ut aut officia dicta omnis quisquam repellendus minus nobis dolorum, doloribus ratione, maxime voluptate eveniet saepe perferendis explicabo illo consectetur nisi! Nihil aspernatur quos iusto saepe vel eveniet ab architecto, quisquam harum! Assumenda magni esse, cupiditate rerum animi possimus minus omnis ullam a fugit molestiae libero veniam iusto impedit consequuntur voluptates at officiis culpa iste minima blanditiis. Nobis porro autem ut maiores facere eos magnam est modi doloremque pariatur eum similique veniam commodi, voluptatem earum, id quasi amet ad consequuntur, sequi eius! Ipsa aliquam fugiat debitis veniam minima autem mollitia praesentium ea id aut ullam vitae, laboriosam ratione sint libero delectus corrupti dolor blanditiis asperiores culpa facere itaque quae totam! Ipsam aperiam totam ab molestias quibusdam at, maxime alias earum dolorem voluptate quos. Quisquam consectetur fuga tempore ducimus architecto dolorem laborum nulla tenetur, illum voluptate doloremque eos, animi libero assumenda blanditiis distinctio beatae? Necessitatibus adipisci deserunt quia tempora possimus a est minima voluptate nisi doloremque dolore laborum, ab sit et, provident dignissimos voluptas rerum quibusdam porro asperiores unde optio! In.</p>
	<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Natus voluptatibus voluptatum nisi ullam! A facilis, dolorem provident. Voluptates inventore ipsum, enim veniam nihil deleniti. Rem molestiae, necessitatibus quos repudiandae possimus ut aut officia dicta omnis quisquam repellendus minus nobis dolorum, doloribus ratione, maxime voluptate eveniet saepe perferendis explicabo illo consectetur nisi! Nihil aspernatur quos iusto saepe vel eveniet ab architecto, quisquam harum! Assumenda magni esse, cupiditate rerum animi possimus minus omnis ullam a fugit molestiae libero veniam iusto impedit consequuntur voluptates at officiis culpa iste minima blanditiis. Nobis porro autem ut maiores facere eos magnam est modi doloremque pariatur eum similique veniam commodi, voluptatem earum, id quasi amet ad consequuntur, sequi eius! Ipsa aliquam fugiat debitis veniam minima autem mollitia praesentium ea id aut ullam vitae, laboriosam ratione sint libero delectus corrupti dolor blanditiis asperiores culpa facere itaque quae totam! Ipsam aperiam totam ab molestias quibusdam at, maxime alias earum dolorem voluptate quos. Quisquam consectetur fuga tempore ducimus architecto dolorem laborum nulla tenetur, illum voluptate doloremque eos, animi libero assumenda blanditiis distinctio beatae? Necessitatibus adipisci deserunt quia tempora possimus a est minima voluptate nisi doloremque dolore laborum, ab sit et, provident dignissimos voluptas rerum quibusdam porro asperiores unde optio! In.</p>
</body>
<script>
	$(document).ready(function(){

		$(".menu-trig").click(function(){
			$(".menu").slideToggle(function(){
				$(this).toggleClass("menu-expanded").css("display", "");
			});
		});
	})
</script>
</html>