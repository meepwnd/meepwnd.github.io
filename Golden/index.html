<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Golden</title>
	<meta name="description" content="This webpage is made by Sergey Bombin using a free PSD template for demonstration purposes only.">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<link rel="icon" href="favicon.ico" type="image/x-icon">
	<link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
	<!-- google fonts -->
	<link href="https://fonts.googleapis.com/css?family=Montserrat:,400,400i,700,700i,900,900i|
	Roboto+Slab:400,700" rel="stylesheet">
	<!-- css -->
	<link rel="stylesheet" href="css/style.css">
</head>
<body>
	<nav class="nav" id="nav">
		<div class="container flex-container">
			<a href="#" class="nav-logo">
				Golden
			</a>
			<ul class="nav-menu">
				<li class="nav-menu_item"><a href="#scrollspy-home">home</a></li>
				<li class="nav-menu_item"><a href="#scrollspy-services">services</a></li>
				<li class="nav-menu_item"><a href="#scrollspy-portfolio">portfolio</a></li>
				<li class="nav-menu_item"><a href="#scrollspy-about">about</a></li>
				<li class="nav-menu_item"><a href="#scrollspy-contact">contact</a></li>
			</ul>

			<div class="nav-menu-trigger">
				<span class="bar1"></span>
				<span class="bar2"></span>
				<span class="bar3"></span>
			</div>
		</div>
	</nav>

	<!-- HERO -->
	<section class="hero" id="scrollspy-home">
		<h2>Welcome To Our Studio!</h2>
		<h1>It's nice to Meet you</h1>
		<a href="#" class="hero-btn"><span>Tell Me More</span></a>
	</section>
	
	<!-- SERVICES -->
	<section class="services" id="scrollspy-services">
		<h2>Services</h2>
		<h6>Proin iaculis purus consequat sem cure.</h6>
		<div class="container flex-container">
			<div class="col-3">
				<div class="services-icon e-commerce">
				</div>
				<h5>E-Commerce</h5>
				<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
			</div>
			<div class="col-3">
				<div class="services-icon responsive">
				</div>
				<h5>Responsive Web</h5>
				<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
			</div>
			<div class="col-3">
				<div class="services-icon security">
				</div>
				<h5>Web Security</h5>
				<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
			</div>
		</div>
	</section>

	<!-- PORTFOLIO -->
	<section class="portfolio" id="scrollspy-portfolio">
		<h2>Our portfolio</h2>
		<h6>Proin iaculis purus consequat sem cure.</h6>
		<div class="container flex-container">
			<figure>
				<img src="images/img1.jpg" alt="">
				<figcaption>
					<h6>Ebony & Ivory</h6>
					<p>Branding</p>
				</figcaption>
			</figure>
			<figure>
				<img src="images/img2.jpg" alt="">
				<figcaption>
					<h6>Smart Stationary</h6>
					<p>Print Design</p>
				</figcaption>
			</figure>
			<figure>
				<img src="images/img3.jpg" alt="">
				<figcaption>
					<h6>Clever Poster</h6>
					<p>Print Design</p>
				</figcaption>
			</figure>
			<figure>
				<img src="images/img4.jpg" alt="">
				<figcaption>
					<h6>Vinyl Record</h6>
					<p>Product Mock-Up</p>
				</figcaption>
			</figure>
			<figure>
				<img src="images/img5.jpg" alt="">
				<figcaption>
					<h6>Treehouse Template</h6>
					<p>Web Design</p>
				</figcaption>
			</figure>
			<figure>
				<img src="images/img6.jpg" alt="">
				<figcaption>
					<h6>Burned Logo</h6>
					<p>Branding</p>
				</figcaption>
			</figure>
		</div>
	</section>

	<!-- ABOUT -->
	<section class="about" id="scrollspy-about">
		<h2>About Us</h2>
		<h6>Proin iaculis purus consequat sem cure.</h6>
		<div class="container">
			<div class="flex-container beginning">
				<div class="about-left">
					<h5><span>July 2010</span><br> Our Humble Beginnings</h5>
					<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit.</p>
				</div>
				<div class="about-img">
					<div class="about-img_beginning"></div>
				</div>
				<div class="about-right mobile-hide"></div>
			</div>

			<div class="flex-container">
				<div class="about-left mobile-hide"></div>
				<div class="about-img">
					<div class="about-img_battles"></div>
				</div>
				<div class="about-right">
					<h5><span>January 2011</span> <br>Facing Startup Battles</h5>
					<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit aenean 
rhoncus posuere odio in tincidunt.</p>
				</div>
			</div>

			<div class="flex-container">
				<div class="about-left">
					<h5><span>December 2012</span> <br> Enter The Dark Days</h5>
					<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni.</p>
				</div>
				<div class="about-img">
					<div class="about-img_dark"></div>
				</div>
				<div class="about-right mobile-hide"></div>
			</div>

			<div class="flex-container">
				<div class="about-left mobile-hide"></div>
				<div class="about-img">
					<div class="about-img_triumph"></div>
				</div>
				<div class="about-right">
					<h5><span>february 2014</span> <br>Our Triumph</h5>
					<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit aenean.</p>
				</div>
			</div>

			<div class="flex-container">
				<div class="about-left"></div>
				<div class="about-center">
					<div class="about-img_story">
						Our<br>
						Story<br>
						Continues<br>
						...
					</div>
				</div>
				<div class="about-right"></div>
			</div>
		</div>
	</section>

	<!-- TEAM -->
	<section class="team">
		<h2>our amazing team</h2>
		<h6>Proin iaculis purus consequat sem cure.</h6>
		<div class="container flex-container">
			<figure>
				<img src="images/img-team.png" alt="">
				<figcaption>
					<h6>Kimberly Thompson</h6>
					<p>Marketer</p>
				</figcaption>
				<div class="social flex-container">
					<a href="#" class="social-link tweeter"><i></i></a>
					<a href="#" class="social-link facebook"><i></i></a>
					<a href="#" class="social-link pinterest"><i></i></a>
				</div>
			</figure>
			<figure>
				<img src="images/img-team.png" alt="">
				<figcaption>
					<h6>Rico Massimo</h6>
					<p>Coder</p>
				</figcaption>
				<div class="social flex-container">
					<a href="#" class="social-link tweeter"><i></i></a>
					<a href="#" class="social-link facebook"><i></i></a>
					<a href="#" class="social-link google"><i></i></a>
				</div>
			</figure>
			<figure>
				<img src="images/img-team.png" alt="">
				<figcaption>
					<h6>Uku Mason</h6>
					<p>Graphic Designer</p>
				</figcaption>
				<div class="social flex-container">
					<a href="#" class="social-link facebook"><i></i></a>
					<a href="#" class="social-link pinterest"><i></i></a>
				</div>
			</figure>
		</div>
		<p class="team-slogan">Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit  aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
	</section>

	<!-- CLIENTS -->
	<section class="clients">
		<div class="clients-inner">
			<img src="images/client1.jpg" alt="">
			<img src="images/client2.png" alt="">
			<img src="images/client3.jpg" alt="">
			<img src="images/client4.jpg" alt="">
		</div>
	</section>

	<!-- CONTACT -->
	<section class="contact" id="scrollspy-contact">
		<h2>Contact us</h2>
		<h6>Proin iaculis purus consequat sem cure.</h6>
		<!-- contact form -->
		<div class="container form-wrapper">
			<form action="">
				<div class="flex-container">
					<div class="col-2">
						<input type="text" placeholder="your name *">
						<input type="email" placeholder="your e-mail *">
						<textarea name="subject" id="subject" placeholder="subject"></textarea>
					</div>
					<div class="col-2">
						<textarea name="message" id="message" placeholder="your message *"></textarea>
					</div>
				</div>
				<input type="button" id="submit" value="send message">
			</form>
		</div>
	</section>

	<!-- FOOTER -->
	<footer class="container flex-container">
		<div class="col-3">
			<p class="footer-copyright">Â© Copyright 2014 FreebiesXpress.com</p>
		</div>
		<div class="social flex-container col-3">
			<a href="#" class="social-link tweeter"><i></i></a>
			<a href="#" class="social-link facebook"><i></i></a>
			<a href="#" class="social-link pinterest"><i></i></a>
			<a href="#" class="social-link google"><i></i></a>
		</div>
		<div class="col-3"></div>
	</footer>

	<!-- SCRIPTS -->
	<script src="js/jquery-3.2.1.min.js"></script>
	<script src="js/jquery.js"></script>

</body>
</html>